### ISSUE TITLE
Empty button false positive

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1198]
This code is incorrectly flagging an empty button error.

```
<button class="brz-btn brz-btn-submit brz-css-3idump brz-css-ci15l0" data-brz-custom-id="h9rx1p5viNpH"><svg class="brz-icon-svg align-[initial] brz-css-e3c1qb brz-css-1pda2xt"><use href="/brizy/wp-content/plugins/brizy/public/editor-build/prod/editor/icons/glyph/tail-right.svg#nc_icon"></use></svg><span data-brz-translate-text="1" class="brz-span brz-text__editor">Submit</span></button>
```

**Comment by amberhinds** [comment]
Here's another example that is flagging empty button but should not:
```
<button type="submit"> <span class="tcb-sf-button-icon"> <span class="thrv_wrapper thrv_icon tve_no_drag tve_no_icons tcb-icon-inherit-style tcb-icon-display" data-css=""><svg class="tcb-icon" viewBox="0 0 512 512" data-id="icon-search-regular"><path d="M508.5 468.9L387.1 347.5c-2.3-2.3-5.3-3.5-8.5-3.5h-13.2c31.5-36.5 50.6-84 50.6-136C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c52 0 99.5-19.1 136-50.6v13.2c0 3.2 1.3 6.2 3.5 8.5l121.4 121.4c4.7 4.7 12.3 4.7 17 0l22.6-22.6c4.7-4.7 4.7-12.3 0-17zM208 368c-88.4 0-160-71.6-160-160S119.6 48 208 48s160 71.6 160 160-71.6 160-160 160z"></path></svg></span> </span> <span class="tve_btn_txt">Search</span> </button>
```

**Comment by amberhinds** [comment]
Also, this is visually hidden and seems like it should not flag:
~<input type="submit" style="display: none !important;">~

**Comment by amberhinds** [comment]
Oh, that's interesting! Odd that they chose to add visibility:hidden to the screen reader text and not just hide the button itself on larger screens. Thanks for the clarification.
